CHANGELOG.unreleased.md

```diff
@@ -6,6 +6,7 @@
 
 ### Released packages
 
+- vhd-lib v0.7.0
```

Comment:
The rule of thumb does not work here :stuck_out_tongue: 

This change depends on the changes we made on `@xen-orchestra/fs`, therefore we should release `vhd-lib` after.

Does that make sense?